src/wallet.cpp

```diff
@@ -1015,6 +1015,193 @@ set<uint256> CWalletTx::GetConflicts() const
     return result;
 }
 
+CAmount CWalletTx::GetDebit(const isminefilter& filter) const
```

Comment:
Yes, thanks for moving this to the .cpp file, all that code doesn't belong in the header in the first place.